frontend/src/diff/Diff.module.css

```diff
@@ -33,6 +33,7 @@
 .source_filename { font-weight: bold }
 .source_function { font-weight: bold; text-decoration: underline }
 .source_other { font-style: italic }
+.source_line_num { font-style: italic }
```

Comment:
This technically isn't needed. With `show_line_numbers=False` there shouldn't be any text formatted with the `source_line_number` format, but I'm including it for completeness.